libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -97,6 +105,38 @@ public static List<VerifiableCredential> filterVCBasedOnProfileType(
         }
     }
 
+    public static List<VerifiableCredential> filterVCBasedOnEvidenceType(
+            List<VerifiableCredential> vcs, EvidenceType... evidenceType) {
+
+        return vcs.stream()
+                .filter(vc -> matchesEvidenceType(vc, Arrays.asList(evidenceType)))
+                .toList();
+    }
+
+    private static boolean matchesEvidenceType(
```

Comment:
Yea, true, the `EvidenceType` is not well defined, part of the reason for this method, as there are two different types currently that relates to fraud - hence the support for list of evidence types.  Was hoping this would be useful in filtering VCs based on evidence type.

Reply:
Ok sure, we can revisit later if need be.